cli/azd/resources/scaffold/templates/resources.bicept

```diff
@@ -407,6 +457,12 @@ module {{bicepName .Name}} 'br/public:avm/res/app/container-app:0.8.0' = {
             secretRef: 'mongodb-url'
           }
           {{- end}}
+          {{- if .DbCosmos}}
+          {
+            name: 'AZURE_COSMOSDB_ENDPOINT'
```

Comment:
Does `AZURE_COSMOS_ENDPOINT` seem any better here?

Reply:
Yes, `AZURE_COSMOS_ENDPOINT` is better.